### ISSUE TITLE
Plugin Requests

### ISSUE THREAD

**Issue by Andrews54757** [AJS-development/OgarUL-Plugin-Library#2]

**Comment by LegitSoulja** [comment]
@Andrews54757 
How do you add more than one command?  

> this.addToHelp[0] = "command1 : command1";
> this.addToHelp[0] = "command2 : command2";

or do we increment addToHelp for each command?

**Comment by Andrews54757** [comment]
to add two commands, change the 0 to any respective number

```
this.addToHelp[0] = "something"
this.addToHelp[1] = "anotherthing"

**Comment by LegitSoulja** [comment]
k

**Comment by Lukkez** [comment]
I have one plugin request (I don't know if already have, i'm just fast on phone to say)
Make a Random-food plugin who the food spawn with random mass like 1 to 50, just like petridish.pw, if add for me it would be a great thing

**Comment by LegitSoulja** [comment]
@Lukkez this is already added. Check the configuration. Food sizes can spawn vary of the percent of food mass grow.

**Comment by LegitSoulja** [comment]
But I like how they made the food on the servers dynamic. The actually move. Only if that could be a plugin @Andrews54757

**Comment by Andrews54757** [comment]
Actualy, i found out that they dont actually move, if you go next to them, even if you dont touch them, you eat them. The moving function is probably just to avoid being sued by miniclip

**Comment by LegitSoulja** [comment]
New plugin idea, i might created.

You can use tampermonkey for the servers. You can create events for ws to get extra keybinds, or send server messages to the client etc. Then you can be able to use special commands for whatever message comes through. That's another big extension.

**Comment by tommyb008** [comment]
A FacebookBot kind of thing if there isnt one already, as this would be the command

minion [playerid] [amount] [name] [spawnmass]

**Comment by LegitSoulja** [comment]
@tommyb008 facebook is not integregated with OgarUl. 

The command also already exist.

**Comment by sethdm02** [comment]
well i was kind of wondering if you could integrate a better auth system were like you can get coins skins etc higher level

**Comment by BLUclanTDS** [comment]
@tommyb008 Go here: https://github.com/AJS-development/OgarUL-Plugin-Library and download super-minions. It has what your looking for

**Comment by LegitSoulja** [comment]
@BLUclanTDS Have you seen the plugin command usage? That's too much. Simply do
> plugin install SuperMinions

**Comment by agariobro12** [comment]
please add Facebook level :) i have an official server

**Comment by agariobro12** [comment]
also level higer=more mass lvl 10 40 start mass

**Comment by LegitSoulja** [comment]
No, we can't support facebook development because we don't have the tools that miniclip uses with facebook. We would have to re-write the store system, and level all over which I doubt won't even happen.

What I have thought about is public databases for servers to store data. Maybe this could be a thing in the future.

**Comment by fayizan** [comment]
When will the ipbots plugin starts to work...?
I have to start bots for particular player each time when the server restarts

**Comment by fayizan** [comment]
bcoz the id reset

**Comment by agariobro12** [comment]
Can you add an plugin To loop 30 min restart plugin?

**Comment by LegitSoulja** [comment]
@agariobro12 yea, what kind of restart? A full restart? or a restart where the map just cleans?

**Comment by agariobro12** [comment]
full

**Comment by agariobro12** [comment]
oh no map just cleans

**Comment by agariobro12** [comment]
and i can Translate all your plugins in german

**Comment by LegitSoulja** [comment]
Actually, no plugin is required to do that xD

**Comment by LegitSoulja** [comment]
You can use the periodic commands feature

**Comment by agariobro12** [comment]
:/

**Comment by LegitSoulja** [comment]
###### Usage:
pcmd [delay] [repeattime] [command] [flag]

**Comment by agariobro12** [comment]
okay and need help an i9 ogar hosting when i uploading the node_modules im got banned i unbanned with ip changer and mac adress changer tool

**Comment by agariobro12** [comment]
any way with no node_modules

**Comment by LegitSoulja** [comment]
You can't use c9 for hosting. You will get banned as soon as they find out.

**Comment by agariobro12** [comment]
ok And mous i need a port forwarding to host an ogar server? and sorry vor my bad english im german

**Comment by LegitSoulja** [comment]
Yes if you are hosting with your computer you will need to port forward your game port(serverPort)

**Comment by agariobro12** [comment]
to see it on the list?

**Comment by agariobro12** [comment]
give´s any tool to do this i don´t have the Router password from my dad

**Comment by agariobro12** [comment]
ah and what i need to port forwarding i have now the password udp and and and

**Comment by CodingKuda** [comment]
@LegitSoulja  I was thinking about a plugin, that allows your to control the console. With the chat..Just a thought.

**Comment by SharkFinPro** [comment]
@CodingKuda That Would be Good, but i would add moderators so you can have it running 24/7 with moderators that can control things through the chat, or just have your friends be able to give themself mass by using commands for practicing tricks and other things

**Comment by SharkFinPro** [comment]
@LegitSoulja How Would I Port forward so other people can play with me? Do I Need To Port Forward? I know My router Paassword, I Know My ipv4 Adress, and My IP Adress.

**Comment by LegitSoulja** [comment]
@SharkFinPro all I can say is to search google based on your specific router on how to port forward.

@CodingKuda there's many options. As of the latest updates there's now a chat command called **rcon**. Inside your settings folder in **config.ini** all the way to the bottom you can specify your password. Whilst in game **chat** you can simple say **/rcon [password] [command]

Another options is to use my plugin I have created called OgarConsole. It runs on a different port w/ authentication.

**Comment by firelightning13** [comment]
I feel like this is minecraft again -_-

**Comment by LegitSoulja** [comment]
@Andrews54757 How can I send a color packet with customLBEnd? I'm making leaderboardFX and i would want people to use color codes for example &aHey = Green(Hey) etc.. Is it possible? Or should i just use the raw Leaderboard instead of customlbend?

**Comment by Agarian** [comment]
i can't play minecraft, my computer has windows 10

**Comment by Agarian** [comment]
but win 10 seems cool

**Comment by LegitSoulja** [comment]
@Agarian You can play minecraft on windows 10... There's also a windows 10 edition of minecraft. Windows 10 sucks, don't get fooled.

**Comment by Agarian** [comment]
Well my computer that has win 7 died, and its not like i can just downgrade to win 7. Even if i could, i don't want to.

**Comment by LegitSoulja** [comment]
You can downgrade, just saying.

**Comment by GetRektAgario** [comment]
What coding languages do you need to know to start coding these plugins?

**Comment by LegitSoulja** [comment]
Haha what? I'll let you guess that one

**Comment by LegitSoulja** [comment]
Check out the wiki

**Comment by GetRektAgario** [comment]
@LegitSoulja you know the ChatMessages plugin you made, would it be possible to send a link via the chat? and how would you do this if it is possible?

**Comment by LegitSoulja** [comment]
? There's a plugin install usage for OgarUl to install plugins automatically without downloading them. What's the point of sending them a direct link?

**Comment by LegitSoulja** [comment]
I was actually working on this > http://ogarul.tk/plugins/ > but never finished. I'll try to finish it asap.

**Comment by LegitSoulja** [comment]
@GetRektAgario ohh, i know you what mean. Hmm, idk actually. Since the text is appended to the canvas I don't really think canvas can handle (selected) text, nor links since its just pixels(text) drawn on a canvas.

**Comment by GetRektAgario** [comment]
@LegitSoulja are you sure or could you make a plugin which pops up words into the top of the screen so one people notice them more and 2 maybe we could add links?

**Comment by LegitSoulja** [comment]
Use the fuck miniclip ad plugin... Thats your only options as of right now.

**Comment by LegitSoulja** [comment]
> plugin install Ads
